resources/configure.rb

```diff
@@ -111,7 +111,7 @@
     end
   end
 
-  directory new_resource.path_data do
+  directory new_resource.path_logs do
```

Comment:
Does that mean we don't need path_data anymore?

Reply:
works as expected on centos7:
```
[root@es-masternode-centos-7 ~]# grep path /etc/elasticsearch/elasticsearch.yml
path.data: "/var/lib/elasticsearch"
path.logs: "/var/log/elasticsearch"
```

Reply:
Excellent, probably a typo on my part then!